apps/web/components/eventtype/EventAvailabilityTab.tsx

```diff
@@ -87,14 +87,18 @@ const EventTypeScheduleDetails = memo(
       { enabled: !!scheduleId || !!loggedInUser?.defaultScheduleId || !!selectedScheduleValue }
     );
 
+    const weekStart = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"].indexOf(
+      loggedInUser?.weekStart || "Sunday"
+    ) as 0 | 1 | 2 | 3 | 4 | 5 | 6;
+
```

Comment:
We can move this logic to util. so, it will get used in sorting function as well
Thanks